### ISSUE TITLE
Build into /usr/local

### ISSUE THREAD

**Issue by nixpulvis** [benben/puppet-rubybuild#1]
I want to install ruby directly to `/usr/local/`, but this module appends the version number to the install path making the install go into a folder inside `/usr/local`.

**Comment by benben** [comment]
hey @nixpulvis, thanks for reporting! will fix this asap!

**Comment by benben** [comment]
hi @nixpulvis,

I released 0.0.4 just now. Its up on puppetforge, so you just need to update it. The `$install_dir` parameter only tells where ruby-build will be installed. You can specify the directory where ruby should be installed with `$ruby_install_dir`. If you call the module like this

```
class { "rubybuild":
  ruby_install_dir => "/usr/local/",
  version_in_dir   => false,
}
```

ruby will install directly to `$ruby_install_dir` without appending the ruby version. The ruby bin will then be at `/usr/local/bin/ruby`. Can you please try it and report back? Thanks!

**Comment by nixpulvis** [comment]
Will do I'll try it out soon.
